### ISSUE TITLE
Query about attributes of links in network.xml

### ISSUE THREAD

**Issue by yangxiong11** [matsim-org/matsim-code-examples#476]
Hello,

In the network.xml file of the "equil" example, I found the "length" of a link is not equal to the linear distance between the start node and the end note. I thought this is because a link represents a road section in reality, which is not a straight line in most cases (may be a curve), and thus the actual length of a link specified by users should be not shorter than the linear distance between two nodes. Am I right? please correct me if I am wrong and give me more explanations. Many thanks!

**Comment by mrieser** [comment]
You are correct. The length of a link should be at least the Euclidean distance between its nodes, but it can be (and often is) longer than that, exactly for the reasons you stated (actual road can be a curve).

MATSim will warn you if a link's length is shorter than the euclidean distance between its nodes, but not if it is longer.
